apps/family-public-law/fpl-case-service/fpl-case-service.yaml

```diff
@@ -8,12 +8,19 @@ spec:
   releaseName: fpl-case-service
   values:
     java:
-      replicas: 2
+      replicas: 4
+      memoryRequests: '1536Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '2000m'
       readinessDelay: 45
       readinessTimeout: 5
       readinessPeriod: 15
       autoscaling:
-        enabled: false
+        enabled: true
+        maxReplicas: 6
```

Comment:
I'd expect this to be 4. Is there a specific reason maxReplicas has been set to 6?